llvm/lib/Target/AMDGPU/SILowerControlFlow.cpp

```diff
@@ -951,6 +945,11 @@ bool SILowerControlFlow::runOnMachineFunction(MachineFunction &MF) {
   optimizeEndCf();
 
   if (LIS) {
+    if (Changed) {
+      // These will need to be recomputed for insertions and removals.
+      LIS->removeAllRegUnitsForPhysReg(AMDGPU::EXEC);
```

Comment:
exec is reserved and has no liveness tracking, not sure why it was handled before